backend/app/api/v1/repositories.py

```
@@ -0,0 +1,24 @@
+
+@router.post("/repo-stats")
+async def analyze_repository(request: RepoRequest):
+    print(f" repo aa gayi : {request.repo_url}")
+
+    try:
+        result = await RepoService().index_repo(request.repo_url)
+        return {
+            "message": "Repository indexed successfully",
+            "repository": request.repo_url,
+            "stats": result
+        }
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)